lib/advert/ui/pages/detail_page/detail.dart

```diff
@@ -19,6 +19,7 @@ class AdvertDetailPage extends HookConsumerWidget {
     final advertPosters = ref.watch(advertPostersProvider);
     final advertPostersNotifier = ref.watch(advertPostersProvider.notifier);
     final logoNotifier = ref.watch(advertPosterProvider.notifier);
+    final filteredTagList = advert.tags.where((element) => element != "").toList();
```

Comment:
Can't you add the advert.announcer.name inside as well ?

Reply:
You are right, it's because we may want to use different widgets for announcer(s) and tags in the futur. For now, I can use a single variable.